src/basic/env-file.c

```diff
@@ -637,3 +637,12 @@ int write_env_file(int dir_fd, const char *fname, char **headers, char **l) {
         (void) unlinkat(dir_fd, p, 0);
         return r;
 }
+
+int write_vconsole_conf(int dir_fd, const char *fname, char **l) {
+        char **headers = STRV_MAKE(
+                "# Written by systemd-localed(8) or systemd-firstboot(1), read by systemd-localed",
+                "# and systemd-vconsole-setup(8). It is probably wise not to edit this file manually."
```

Comment:
it' still drop the "It is probably wise not to edit this file manually". It should be fine to edit it manually, if people want. I think you can strike this and just keep the recommendation to use localectl(1). It alone conveys the suggestion that that's the primary interface to it.